hutch_python/tests/test_happi.py

```diff
@@ -34,6 +34,25 @@ def test_happi_objs():
             lc = get_lightpath(tmp.name, 'tst')
 
 
+@pytest.mark.parametrize('load_level, num_devices', [
+    (DeviceLoadLevel.UPSTREAM, 3),
+    (DeviceLoadLevel.STANDARD, 4),
+    (DeviceLoadLevel.ALL, 5)
```

Comment:
These don't quite match my understanding of the levels.
Upstream - I agree it should be 3 since that's all the TST devices
Standard - shouldn't this be 5, including devices 4 and 5 on X0?
All - shouldn't this be 6, including the Z0 device?

Reply:
I think the updated map I posted should clear this up, but the main difference is that `tst_device_3` is inactive, and isn't seen